src/main.zig

```diff
@@ -213,6 +213,28 @@ pub const std_options: std.Options = .{ // MARK: std_options
 	}.logFn,
 };
 
+/// The maximum size a log message can have.
+const log_buffer_size = 64 << 10;
+
+pub fn panicToLog(msg: []const u8, first_trace_address: ?usize) noreturn {
+	const addr = first_trace_address orelse @returnAddress();
+	std.log.err("This is probably a bug, if you think so please make and issue and add the entire log: <https://github.com/pixelguys/cubyz/issues/new?template=bug.yml>\n----8<---- start of panic\npanic: {s}\nerror return trace: {?f}", .{
```

Comment:
```suggestion
	std.log.err("This is probably a bug. If you think so, please make an issue and add the entire log: <https://github.com/pixelguys/cubyz/issues/new?template=bug.yml>\n----8<---- start of panic\npanic: {s}\nerror return trace: {?f}", .{
```

Tiny suggestion, but and -> an and slight punctuation changes

Reply:
Thanks! Will apply.